### PR TITLE
Add Unit Tests

### PR DESCRIPTION
**Pull request by nicklera1**
**Description of changes:**
Add UT UserActivityManager 100%

**Testing**
1. Have you successfully run `npm run build:release` locally?

2. How did you test these changes?

3. If you made changes to the component library, have you provided corresponding documentation changes?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tst/components/ui/UserActivityManager/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    return (
+      <UserActivityContext.Provider value={state}>
+        <UserActivityManager>
+          <h3>{testString}</h3>
+        </UserActivityManager>
+      </UserActivityContext.Provider>
+    );
+  };
+
+  it('should render a UserActivityManager when user isActive', () => {
```
Comment (ziyiz-amzn) [inline tst/components/ui/UserActivityManager/index.test.tsx:30]: nit: `isUserActive=true`

File: tst/components/ui/UserActivityManager/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import '@testing-library/jest-dom';
+import React, { useState } from 'react';
+import { fireEvent } from '@testing-library/dom';
```
Comment (ziyiz-amzn) [inline tst/components/ui/UserActivityManager/index.test.tsx:6]: nit: unused

File: tst/components/ui/UserActivityManager/index.test.tsx (hunk 1 above)
Comment (nicklera1) [reply tst/components/ui/UserActivityManager/index.test.tsx]: done

File: tst/components/ui/UserActivityManager/index.test.tsx (hunk 2 above)
Comment (nicklera1) [reply tst/components/ui/UserActivityManager/index.test.tsx]: removed

**Review by ziyiz-amzn — APPROVED** [review]

**Pull request merged**